mlrun/datastore/targets.py

```diff
@@ -548,7 +548,7 @@ def write_dataframe(
                     partition_cols + (self.partition_cols or [])
                     if partition_cols
                     else self.partition_cols
-                )
+                ) or []
```

Comment:
Seems unnecessarily complicated. How about:
```suggestion
                partition_cols = partition_cols + (self.partition_cols or [])
```

Reply:
I guess even better
```python
                partition_cols += self.partition_cols or []
```

Reply:
Or
```python
                if self.partition_cols:
                    partition_cols += self.partition_cols
```
if you like that better.